brave/brave-core
# Fix #17873 - Disable media router by default

Resolves https://github.com/brave/brave-browser/issues/17873

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Open brave browser with a clean profile.
2. Open the hamburger menu and verify that cast is not available
3. Verify that SSDP requests are not initiated on start
4. Navigate to brave://settings/extensions - Enable `Media Router` and restart
5. Verify that cast is available and functional.
